MvvmCross/MvvmCross.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461;uap10.0.16299;Xamarin.iOS10;Xamarin.Mac20;Xamarin.TVOS10;Xamarin.WatchOS10;MonoAndroid80;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461;uap10.0.16299;Xamarin.iOS10;Xamarin.Mac20;Xamarin.TVOS10;Xamarin.WatchOS10;MonoAndroid80</TargetFrameworks>
```

Comment:
Why remove this target?

Reply:
The target was _only_ for `MvxIoCSupportingTest.cs`

Reply:
Yes, but we can add more test helpers in there. With that you could install the main MvvmCross lib into any project without pulling in a lot of other dependencies.

Reply:
That doesn't make sense. What if I add a shared Fixture that depends on Xunit? Then all of the sudden MvvmCross depends on Xunit everywhere.

Reply:
With multi target that would only add the dependency in the netcoreapp target.

Reply:
It still doesn't make sense, because if we want to support netcoreapp2.0 in the future, not just use it for testing, we will need to break stuff again.

Reply:
Hmm, right you got a point there. So doesn't the new project need to target netcoreapp then?

Reply:
netstandard there is fine. It is just our own test assemblies that need to be netcoreapp2.0 or net full framework